recipes/cron.rb

```diff
@@ -98,7 +105,7 @@ class ::Chef::Recipe
     user    'root'
     cmd = ''
     cmd << "/bin/sleep #{sleep_time}; " if sleep_time
-    cmd << "#{env} #{client_bin} #{append_log} #{log_file} 2>&1"
+    cmd << "#{env} #{cron_client_bin} #{append_log} #{log_file} 2>&1"
```

Comment:
This solution seems super messy. Why can't we just do something like

```ruby
daemon_options = " #{node["chef_client"]["daemon_options"].join(' ')}" if node["chef_client"]["daemon_options"].any?
...
cmd << "#{env} #{client_bin} #{daemon_options} #{append_log} #{log_file} 2>&1"
```

instead of all the `dup` stuff?